src/main/java/twilightforest/TwilightForestMod.java

```diff
@@ -28,7 +28,7 @@
 		name = "The Twilight Forest",
 		version = TwilightForestMod.VERSION,
 		acceptedMinecraftVersions = "[1.12.2]",
-		dependencies = "after:ctm@[MC1.12-0.2.3.9,);required-after:forge@[14.23.0.2501,)")
+		dependencies = "after:ctm@[MC1.12-0.2.3.9,);required-after:forge@[14.23.0.0,)")
```

Comment:
Absolutely not

Reply:
This change was to allow development versions of Forge to be used with this mod/any 1.12.2 Forge Version.

For this, what do you suggest I do here, my idea was if this isn't allowed, to instead use the Forge Version it's build with here.

Reply:
Forge build `14.23.0.2496` introduces the ability to add custom map icons, we'd prefer to forge the forge version to any version after this or else Magic Maps will render with bad structure icons.

Reply:
Ah, Ok. I see. In that case, ill set the required build to any one after 14.23.0.2496 in the next commit to this PR (Next Commit will update MCP again to Today's Release from Yesterday's)

Meanwhile, are there any other concerns on this PR? (Mainly with the MCP Changes and other dependencies)

Reply:
Right now my next concern is that I will be forced to re-run setup as well as 4 other people, and I'll hold until CTM gets an update.

Reply:
*facepalm* Completly forgot to change this line lol

After this PR, it'll now require Forge 14.23.0.2496 or higher 👍

Reply:
*Proceeds to mess up further by still not changing the line* Will fix this today with MCP Follow up to keep this PR maintained.